scripts/update-version.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python3
+# This script will to help to manage rhpam components modules version, it will update all needed files
+# Example of usage:
+#   # move the current version to the next one or rcX
+#   python scripts/update-version.py -v 7.15.1
```

Comment:
Maybe define `python3` as caller since till fedora 30 python2 is called when someone runs `python` command

Reply:
+1.